webapp/channels/src/plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -194,13 +198,17 @@ class ChannelHeaderPlug extends React.PureComponent<ChannelHeaderPlugProps, Chan
             ariaLabel = tooltipToAriaLabelHack(this.props.intl, plug.dropdownText);
         }
 
+        // TODO: Remove this any and make sure the types are properly
+        // handled.
+        const tooltipText: any = plug.tooltipText ?? plug.dropdownText ?? '';
```

Comment:
I forget why the existing code worked the way it did since I remember being confused why the ARIA label and the tooltip are different. I assume it's something similar to why this still has to use an `any` 😅